### ISSUE TITLE
Create Mekanism Compatibility?

### ISSUE THREAD

**Issue by real-chair** [NillerMedDild/Enigmatica6#1373]
Hey, sorry for the poor formatting; this is my first time using github.
Basically, I was thinking it'd be cool if crushing and washing recipes were added to the Create contraptions for the Mekanism ore clumps and dirty dust, respectively.

Just thought it might be a good idea  because Create becomes quickly obsolete in lieu of other tech mods; letting us mix and match would give us the efficiency of other tech mods with the stunning  visuals of Create. On top of that, this change wouldn't affect much balance-wise, as Create's method of washing dust for 1.6x ore processing only works on it's own custom crushed ores.

TL;DR
Add in Mekanism ore processing compatibility for Create machines pls.
(PS: sorry again for the poorly written post)

**Comment by MuteTiefling** [comment]
Mekanism and Create have mutually exclusive ore processing systems. 

The benefit to Create's processing, besides being very energy efficient, is that it gives you secondary outputs. If I recall properly, the outputs are also on par with about a 3.5x yield per ore.
